### DISCUSSION TITLE
Twig Template Loading Error

### DISCUSSION THREAD

**Discussion by ankitupadhyay360** [solspace/craft-freeform#1272] [Questions & Answers] [answered]
Unable to find the template “freeform/C:\xampp\htdocs\sandyou\vendor\solspace\craft-freeform\packages\plugin\src\templates\_templates\formatting\basic-light\index.twig”.

How To fix This issue

**Comment by kjmartens** [comment, marked as answer]
Hi @ankitupadhyay360,

The path should actually be: ...`\templates\_templates\formatting\basic-light\index.twig`

I wonder if you just need to add a trailing slash on your templates directory path? 🙂
